data/aliases.js

```diff
@@ -637,6 +639,7 @@ exports.BattleAliases = {
 	"se": "Stone Edge",
 	"spin": "Rapid Spin",
 	"sr": "Stealth Rock",
+	"st": "Spectral Thief",
```

Comment:
Does Spectral Thief deserve an alias, considering it's a signature move of one Pokemon, and ST I would rather think is Sleep Talk.